cmd/nitro/init.go

```diff
@@ -326,6 +336,36 @@ func openInitializeChainDb(ctx context.Context, stack *node.Node, config *NodeCo
 		}
 	}
 
+	// Check if database was misplaced in parent dir
+	const errorFmt = "database was not found in %s, but it was found in %s (have you placed the database in the wrong directory?)"
+	parentDir := filepath.Dir(stack.InstanceDir())
+	if dirExists(path.Join(parentDir, "arbitrumdata")) {
```

Comment:
I think that it should be cleaner to check existence of `l2chaindata` directory here instead, as we don't do anything with `arbitrumdata` database in `openInitializeChainDb` - we open only `l2chaindata` and `wasm` databases.

We probably could also end up messaging that the `arbitrumdata` database was not found in the `stack.InstanceDir()`, but we haven't checked that yet.